zerver/openapi/test_curl_examples.py

```diff
@@ -47,21 +48,9 @@ def test_generated_curl_examples_for_success(client: Client, owner_client: Clien
                 unescaped_html = html.unescape(curl_command_html)
                 curl_command_text = unescaped_html[len("<p><code>curl\n") : -len("</code></p>")]
                 curl_command_text = curl_command_text.replace(
-                    "BOT_EMAIL_ADDRESS:BOT_API_KEY", authentication_line
+                    "BOT_EMAIL_ADDRESS:BOT_API_KEY", curl_global_variable.AUTHENTICATION_LINE
                 )
```

Comment:
I'm not sure what'd actually be nice here, but one candidate would be to just support checking if there's an entry for the filename in a dictionary in exported by `curl_param_value_generators.py `, and if so, call that function at the appropriate place in this file.

Reply:
You're suggesting that we detect filename in `curl_param_value_generators.py`, and from there, we call a function in this file. That changes the `authentication_line`. Right?
If yes, for this to work, we have to import that function in `curl_param_value_generators.py`, and I think that not possible because we're already importing variables from `curl_param_value_generators.py`, and this will create [circular import error](https://stackoverflow.com/questions/17845366/importerror-cannot-import-name/17845428).
Even if this will workout, our primary intention was to keep all the endpoint code and `if` statement away from `test_curl_examples.py.`